x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/email.tsx

```diff
@@ -97,26 +97,6 @@ export function getActionType(): ActionTypeModel {
           )
         );
       }
-      if (!action.secrets.user) {
-        errors.user.push(
-          i18n.translate(
-            'xpack.triggersActionsUI.components.builtinActionTypes.error.requiredUserText',
-            {
-              defaultMessage: 'Username is required.',
-            }
-          )
-        );
-      }
-      if (!action.secrets.password) {
-        errors.password.push(
-          i18n.translate(
-            'xpack.triggersActionsUI.components.builtinActionTypes.error.requiredPasswordText',
-            {
-              defaultMessage: 'Password is required.',
-            }
-          )
-        );
-      }
```

Comment:
Maybe it makes sense to do like we have for a webhook:

if (!action.secrets.user && action.secrets.password) {
        errors.user.push(
          i18n.translate(
            'xpack.triggersActionsUI.sections.addAction.webhookAction.error.requiredHostText',
            {
              defaultMessage: 'Username is required.',
            }
          )
        );
      }
      if (!action.secrets.password && action.secrets.user) {
        errors.password.push(
          i18n.translate(
            'xpack.triggersActionsUI.sections.addAction.webhookAction.error.requiredPasswordText',
            {
              defaultMessage: 'Password is required.',
            }
          )
        );
      }

Reply:
makes sense, I'll do that